src/modules/VirtConfig.rb

```diff
@@ -402,51 +378,24 @@ def LXCDialog
       common_vm_packages = []
 
       result = true
-      if isOpenSuse == true
-        if install_vm == true
-          common_vm_packages = ["libvirt-client", "vm-install", "virt-install"]
-        end
-
-        packages = ["patterns-server-xen_server"] if install_xen_server
-        if isTumbleweed
-          packages = packages + ["xen-tools", "xen-libs", "libvirt-daemon-xen", "libvirt-daemon-config-network", "tigervnc", "virt-manager"] if install_xen_tools
-        else
-          packages = ["patterns-server-xen_tools"] if install_xen_tools
-        end
-
-        packages = packages + ["patterns-server-kvm_server"] if install_kvm_server
-        if isTumbleweed
-          packages = packages + ["libvirt-daemon-qemu", "libvirt-daemon-config-network", "tigervnc", "virt-manager"] if install_kvm_tools
-        else
-          packages = ["patterns-server-kvm_tools"] if install_kvm_tools
-        end
-        packages = packages + ["libvirt-daemon-lxc", "libvirt-daemon-config-network"] if install_lxc
-        result = Package.DoInstall(common_vm_packages + packages)
+      if install_lxc
+        packages = ["libvirt-daemon-lxc", "libvirt-daemon-config-network"]
+        result = Package.DoInstall(packages)
         if result == false
-          Report.Error(_("Package installation failed\n"))
+          Report.Error(_("Package installation failed for lxc\n"))
           return false
         end
-      else
-        if install_lxc
-          packages = ["libvirt-daemon-lxc", "libvirt-daemon-config-network"]
-          result = Package.DoInstall(packages)
-          if result == false
-            Report.Error(_("Package installation failed for lxc\n"))
-            return false
-          end
-        end
-        if isSLES
-          packages = packages + ["patterns-server-xen_server"] if install_xen_server
-          packages = packages + ["patterns-server-xen_tools"] if install_xen_tools
-          packages = packages + ["patterns-server-kvm_server"] if install_kvm_server
-          packages = packages + ["patterns-server-kvm_tools"] if install_kvm_tools
-          packages = installGUIComponents(packages)
-          result = Package.DoInstall(packages)
-          if result == false
-            Report.Error(_("Package installation failed for sles patterns\n"))
-            return false
-          end
-        end
+      end
+
+      packages = packages + ["patterns-server-xen_server"] if install_xen_server
+      packages = packages + ["patterns-server-xen_tools"] if install_xen_tools
+      packages = packages + ["patterns-server-kvm_server"] if install_kvm_server
+      packages = packages + ["patterns-server-kvm_tools"] if install_kvm_tools
+      packages = installGUIComponents(packages)
+      result = Package.DoInstall(packages)
+      if result == false
+        Report.Error(_("Package installation failed for sles patterns\n"))
```

Comment:
The `sles patterns` part is not true anymore, now the same code is executed also in Tumbleweed. 

You could also use the generic [Message::FailedToInstallPackages](https://github.com/yast/yast-yast2/blob/master/library/general/src/modules/Message.rb#L258) message here to avoid an additional translation.

Reply:
Pushed this change.